movieApp/asset/app.js

```diff
@@ -0,0 +1,50 @@
+//const API_KEY=daf7b11fd7dfe1a364da7a86d79ba727;
+const IMG_URL="https://image.tmdb.org/t/p/w500";
+const url="https://api.themoviedb.org/3/search/movie?api_key=daf7b11fd7dfe1a364da7a86d79ba727";
+
+//selecting search elements from DOM
+const inputElement=document.querySelector("#searchBox");
```

Comment:
spaces around `=`. You can install Prettier for VSCode and it will do this things automatically.

Reply:
I have prettier installed. I don't known why its not doing things automatically.